frontend/javascripts/oxalis/view/right-border-tabs/connectome_tab/synapse_tree.tsx

```diff
@@ -234,7 +234,7 @@ class SynapseTree extends React.Component<Props, State> {
           <Dropdown // Lazily create the dropdown menu and destroy it again, afterwards
             overlay={() => this.createSegmentDropdownMenu(data.id)}
             autoDestroy
-            placement="bottomCenter"
+            placement="bottom"
```

Comment:
Saw this deprecation warning: `Warning: [antd: Dropdown] You are using 'bottomCenter' placement in Dropdown, which is deprecated. Try to use 'bottom' instead. `

Reply:
I upgraded antd to a slightly newer version, because although this was deprecated, the type annotations didn't allow it yet :see_no_evil: 
As far as I can see, not much changed, but typing seems to have been improved and I was able to remove quite a few type ignores.